specification/web/resource-manager/Microsoft.Web/stable/2024-04-01/WebApps.json

```diff
@@ -6154,6 +6154,52 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/machinekey": {
```

Comment:
>machinekey [](http://example.com/codeflow?start=109&length=10)

consider using an action name like updatemachinekey so that the action is clear to the caller.